tests/helper.py

```diff
@@ -0,0 +1,73 @@
+import os
+import gc
+import logging
+
+
+def init_file_logger(exp_folder: str):
+    LOGGER = logging.getLogger()
+    LOGGER.setLevel(logging.INFO)
+    LOGGER.handlers = []  # removes all handlers
+    formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+    fh = logging.FileHandler(os.path.join(exp_folder, "log.txt"))
+    fh.setFormatter(formatter)
+    LOGGER.addHandler(fh)
+
+
+def remove_previously_created_files(folder: str, extensions_to_delete=("txt", "json")):
```

Comment:
This doesn't seem to be used anywhere.

Reply:
I agree - I pulled it over from another project I was working on and didn't realize it is not needed for this one.  Removed.

Reply:
That is, removed remove_previously_created_files(...)